app/src/main/res/values-fr/strings.xml

```diff
@@ -297,7 +297,7 @@
     <string name="shape_square">Carré</string>
     <string name="shape_teardrop_top_left">Larme haut-gauche</string>
     <string name="shape_teardrop_random">Larme au hasard</string>
-    <string name="shape_squircle">Carré Arrondi</string>
+    <string name="shape_squircle">Carré arrondi</string>
```

Comment:
Another occurrence of unusual title casing (plus other items in this list are not in title case).